lib/words.rb

```diff
@@ -0,0 +1,31 @@
+class Words
+
+  def get_words()
+    words.map {|key, value| value}
```

Comment:
Why do you need a key/value map with `:word_1` etc if you only use the value side?
An array would serve the same purpose.
And the same "get_" prefix we don' t traditionally use 😺

Reply:
I don't know, I was trying to use a module here but without success...

Reply:
Its ok. It's always a pleasure to simplify code 🛩